src/Rings/mpoly-localizations.jl

```diff
@@ -2489,9 +2781,21 @@ julia> UT = complement_of_point_ideal(T, [0]);
 
 julia> TL, _ =  localization(T, UT);
 
-julia> PHI = hom(RQL, TL, TL.([t, t^2, t^3]));
-
-julia> PSI = hom(TL, RQL, RQL.([x]));
+julia> PHI = hom(RQL, TL, TL.([t, t^2, t^3]))
+Ring homomorphism
+  from Localization of quotient of multivariate polynomial ring at complement of maximal ideal
+  to   localization of multivariate polynomial ring in 1 variable over QQ at complement of maximal ideal
```

Comment:
for MPolyQuo the localizations start with a capital letter, for MPolyRing with a lowercase letter. Probably there is still a typo in the MPolyQuo-case somewhere.

Reply:
there is still something fishy in our printing system. @thofma  is investigating